source/include/core_mqtt_config_defaults.h

```diff
@@ -100,6 +100,12 @@
  * If a ping response is not received before this timeout, then
  * #MQTT_ProcessLoop will return #MQTTKeepAliveTimeout.
  *
+ * @note If a dummy implementation of the #MQTTGetCurrentTimeFunc_t timer function,
```

Comment:
1. Let's just say that if a dummy implementation of the timer function is used. I don't we need to go into details of 'that always returns zero'. Let's keep it simple.
2.  processLoop is not supported without a 'real' time function. In that case receiveLoop should be used. ( receive loop is exactly processLopp mius keep alive.

Reply:
Have incorporated point 1. The second point is not appropriate for this config that is only utilized by ProcessLoop.